djedi/plugins/list.py

```diff
@@ -8,14 +8,14 @@
 class ListPlugin(DjediPlugin):
     ext = 'list'
 
-    def get_editor_context(self, **kwargs):
+    def get_editor_context(self, request, **kwargs):
```

Comment:
Might as well follow naming of `DjediPlugin`
```suggestion
    def get_editor_context(self, request, **context):
```